person.rb

```diff
@@ -0,0 +1,28 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    return false unless is_of_age? || @parent_permission
+
+    true
+  end
+
+  private
+
+  def of_age?
+    return true if @age >= 18
+
+    false
```

Comment:
- [ ] Good check :+1: . Ruby is clean in terms of syntax. Simplicity makes it a gem  `@age >= 18` :gem: